Packs/ArcherRSA/Integrations/integration-ArcherRSA.yml

```diff
@@ -2,11 +2,11 @@ commonfields:
   id: RSA Archer
   version: -1
 name: RSA Archer
-display: RSA Archer
+display: RSA Archer - Deprecated
```

Comment:
@ChanochShayner does this match the standard we're enforcing with your new dev task?

Reply:
No, The display_name (display) of all deprecated integrations should end with (Deprecated).

Reply:
@adi88d @yaakovi FYI

Reply:
Also, the description of all deprecated integrations should start with "Deprecated."